pages/print_labels/ad_hoc_plate.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <b-container>
+    <b-row>
+      <b-col>
+        <h1>Print Control plate labels</h1>
+        <p class="lead"></p>
+
+        <!-- TODO: better in a component of its own? -->
```

Comment:
There is an Alert component, which might be useful instead. For an example it's being implemented in the Beckman Cherrypick view